iron-list.html

```diff
@@ -989,6 +1064,139 @@
           this.scrollToIndex(this.firstVisibleIndex);
         }
       });
+    },
+
+    _getModelFromItem: function(item) {
+      var key = this._collection.getKey(item);
+      var pidx = this._physicalIndexForKey[key];
+
+      if (pidx !== undefined) {
+        return this._physicalItems[pidx]._templateInstance;
+      }
+      return null;
+    },
+
+    /**
+     * Select the list item at the given index.
+     *
+     * @method selectItem
+     * @param {Object} item the item object or its index
+     */
+    selectItem: function(item) {
+      if (typeof item === 'number') {
+        item = this.items[item];
+        if (!item) {
+          throw new RangeError('<item> not found');
+        }
+      } else {
+        if (this._collection.getKey(item) === undefined) {
+          throw new TypeError('<item> should be a valid item');
+        }
+      }
+
+      var model = this._getModelFromItem(item);
+
+      if (this.selected && !this.multi) {

```

Comment:
I thought about this and I believe we could move this behavior to `toggleSelection`.  The `toggle` attribute could be used in combination with `selectionEnabled` for tap events. I updated the description for this property, but let me know what you think.